.github/workflows/cache.yml

```
@@ -0,0 +1,36 @@
+name: Cache Keyman API
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Fetch from Keyman API
+        run: |
+          curl https://api.keyman.com/search?q= > src/app/output.json
```

Let's use a more descriptive name than `output.json` 😂  😂  😂 

```suggestion
          curl https://api.keyman.com/search?q= > src/gen/cached-keyman-api.json
```

Why the `gen` folder? I'm using the convetion of [Android](https://developer.android.com/studio/projects#ProjectView) to place generated files in `gen` instead of with the other source code.